src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -1190,6 +1280,60 @@ void MenuGame::draw()
     {
         window.setMouseCursorVisible(hg::Config::getMouseVisible());
     }
+
+    if(!dialogText.empty())
+        drawDialogBox();
+}
+
+void MenuGame::drawDialogBox()
```

Comment:
I strongly feel like all of the dialog box related data members and member functions should be encapsulated in a `DialogBox` class.

Reply:
I totally agree, I will take care of it